lib/data/hive/hive_collection_tom_database.dart

```diff
@@ -29,7 +30,11 @@ class HiveCollectionToMDatabase {
     HiveAesCipher? hiverCipher;
     try {
       // Workaround for secure storage is calling Platform.operatingSystem on web
-      if (kIsWeb) throw MissingPluginException();
+      if (kIsWeb) {
+        // ignore: unawaited_futures
+        html.window.navigator.storage?.persist();
```

Comment:
@dab246 also questioned about it. Can you explain something?

Reply:
If you call this you want your app not to be automatically cleared when storage is low and you can just remove it manually by going to browser settings. 
By default, a site's data `(IndexedDB, Cache API,...)` falls into the best effort category, the browser may evict site data at its discretion, for example when device storage is low.